python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -695,7 +709,7 @@ def export(method):
     return method
 
 
-class ObjectMetaclass(type):
+class ObjectMetaclass(type):#, object):
```

Comment:
I was toying with something at the time and this can be removed to avoid un-needed conversations about how this doesn't work.